src/lib.rs

```diff
@@ -428,6 +428,22 @@ impl Build {
         self
     }
 
+    /// removes a compiler flag that was added by [`Build::flag`]
```

Comment:
Minor doc suggestion:

```suggestion
    /// Removes a compiler flag that was added by [`Build::flag`]
```